Tests/test_PDB.py

```diff
@@ -1577,6 +1577,22 @@ def test_pdb_to_xyzr(self):
         self.assertSequenceEqual(msms_radii, biopy_radii)
 
 
+class FragmentMapperTests(unittest.TestCase):
+    """Tests for FragmentMapper module."""
+    
+    def self_test(self):
+        """Self test for FragmentMapper module."""
+        p = PDBParser()
+        pdb1 = "PDB/1A8O.pdb"
+        s = p.get_structure("X", pdb1)
+        self.assertEqual(m, s[0])
```

Comment:
You can't compare ``m`` to ``s[0]`` if ``m`` has never been defined.

The original code was ``m = s[0]`` which makes more sense here - defining ``m`` as the first model in the structure (index zero).